GOVERNANCE.md

```diff
@@ -0,0 +1,413 @@
+# Magic Modules Governance (a.k.a. "Golden Rules")
+
```

Comment:
As a preamble, I'd suggest something like,

> Please refer to the top-level [README](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/README.md) and [Philosophy](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/docs/philosophy.md) before reading this guide. All code-reviews of Magic Modules (regardless of changes to core, providers, or products) must follow the guidelines outlined here.

Reply:
Should this be instead of the first paragraph, or in addition to?